website/content/userguide/bluetooth.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Using Bluetooth"
+menuTitle: "Using Bluetooth"
+weight: 40
+aliases:
+  - /userguide/bluetooth/
+---
+
+gokrazy has limited support for Bluetooth. The usual Bluetooth utilities (bluez)
+have a lot of dependencies that are not available on gokrazy and those are out
+of scope here. But the basic Bluetooth hardware are supported on low level. This
+is useful in particular for applications using Bluetooth LE natively in Go.
+
+To enable Bluetooth in gokrazy, first include the
+`github.com/gokrazy/bluetooth` package in your `gokr-packer` command line, e.g.:
+
+```shell
+gokr-packer \
+  -tls=self-signed \
+  -update=yes \
+  github.com/gokrazy/hello \
+  github.com/gokrazy/bluetooth
+```
+
+The `github.com/gokrazy/bluetooth` package loads the appropriate kernel modules
+and firmware required.
```

Comment:
Technically there's no firmware required… But the package should load firmware if it was required/beneficial. That it doesn't now, maybe technical detail and it's ok to say here it loads the firmware required, which currently is nothing?

Reply:
Filed https://github.com/gokrazy/bluetooth/issues/2 so that we don’t forget. Yeah, I think it’s okay to state it here already.